.eslintrc.js

```diff
@@ -194,9 +185,30 @@ module.exports = {
       },
     },
     {
-      files: ["*.spec.js"],
+      env: { jest: true },
+      files: ["frontend/test/unit/**"],
+      plugins: ["jest"],
+      extends: ["plugin:jest/recommended"],
       rules: {
+        "import/no-named-as-default-member": ["off"],
         "@intlify/vue-i18n/no-raw-text": ["off"],
+        "no-restricted-imports": [
+          "error",
+          {
+            name: "pinia",
+            message:
+              "Please import pinia test utils from `~~/test/unit/test-utils/pinia`. The test-utils version ensures proper setup of universally necessary Nuxt context mocks.",
+          },
+        ],
+        "no-restricted-syntax": [
+          "error",
+          {
+            selector:
+              "ImportDeclaration[source.value='@vue/test-utils']:has(ImportSpecifier[local.name='shallowMount'])",
+            message:
+              "Do not use @vue/test-utils' `shallowMount`. Use `~~/test/unit/test-utils/render` instead which includes helpful context setup or @testing-library/vue's `render` directly.",
+          },
+        ],
       },
```

Comment:
Using a single ESLint configuration with `overrides` to configure sub-folder behaviour is ultimately easier to understand, I think, than nested configurations. We also had incorrectly configured the nested configuration with an unnecessary `extends` setting, (which causes a new configuration to be created rather than a merged one), showing that it's harder to maintain. It also makes it easier to reason about which rules apply where and which ones are shared without having to jump between multiple files. We can move the other ESLint configuration files into overrides in separate PRs.